pkg/clusteragent/admission/mutate/common.go

```diff
@@ -66,7 +66,7 @@ func injectEnv(pod *corev1.Pod, env corev1.EnvVar) bool {
 			log.Debugf("Ignoring container '%s' in pod %s: env var '%s' already exist", ctr.Name, podStr, env.Name)
 			continue
 		}
-		pod.Spec.Containers[i].Env = append(pod.Spec.Containers[i].Env, env)
+		pod.Spec.Containers[i].Env = append([]corev1.EnvVar{env}, pod.Spec.Containers[i].Env...)
```

Comment:
Note:  this has the effect of reversing the order of our injected env vars, since each is now prepended before the previous one.  This is why the tests have changed.

Reply:
Could you add a comment to explain why we prepend? (can be added in the function doc string as well).